README.md

```diff
@@ -1,15 +1,24 @@
 [![Build Status](https://travis-ci.com/tarantool/metrics.svg?branch=master)](https://travis-ci.com/tarantool/metrics)
+
 # Metrics
 
-Metrics is a tool to collect and manipulate metrics time series.
-Metrics uses a collection of primitives borrowed from the Prometheus TSDB,
-which can be exported to any TSDB.
+Metrics is a library to collect and expose [Tarantool](https://tarantool.io)-based applications metrics.
+
+Library includes:
+- four base metric collectors: Counter, Gauge, Histogram, Summary
+- ready to use Tarantool stats collectors built on top of base collectors
+- exporters to expose collected metrics in Prometheus, Graphite and generic JSON format
+- module to integrate into Tarantool Cartridge based applications
 
-Contents:
+## Table of contents
 
 * [Installation](#installation)
-* [Plugins export](#plugins-export)
-* [Examples](#examples)
+* [Exporters](#plugins-export)
```

Comment:
This section's title is different in TOC and in text (Exporters vs Plugins export). It's confusing. Would be nice to choose a single title (whichever you like).

Reply:
good point, done